game/src/main/resources/assets/sol_client/lang/fi_FI.lang

```diff
@@ -0,0 +1,442 @@
+sol_client.mod.reach_display.description=Näytä etäisyys osuessaan entiteettiin.
+
+sol_client.mod.combo_counter.name=Combon Laskija
+sol_client.mod.combo_counter.description=Laske peräkkäisien osumien määrä.
+
+sol_client.mod.combo_counter.no_hits=Ei comboa
+sol_client.mod.combo_counter.one_hit=1 osuma
+sol_client.mod.combo_counter.n_hits=%s osumaa
+
+sol_client.mod.potion_effects.name=Näytä Taikajuomat
+sol_client.mod.potion_effects.description=Näytä aktiiviset taikajuomat laskimella.
+
+sol_client.mod.potion_effects.option.alignment=Linjaus
+sol_client.mod.potion_effects.option.icon=Kuva
+sol_client.mod.potion_effects.option.background=Tausta
+sol_client.mod.potion_effects.option.title=Etiketti
+sol_client.mod.potion_effects.option.titleColour=Etiketin Väri
+sol_client.mod.potion_effects.option.duration=Aika
+sol_client.mod.potion_effects.option.durationColour=Ajan Väri
+sol_client.mod.potion_effects.option.spacing=Välitys
+
+sol_client.mod.armour.name=Panssari ja Varusteet
+sol_client.mod.armour.description=Näytä panssarisi ja varusteet käyttömäärän kanssa.
+
+sol_client.mod.armour.option.durability=Käyttömäärän Teksti
+sol_client.mod.armour.option.durability.off=Pois
+sol_client.mod.armour.option.durability.remaining=Jäljellä
+sol_client.mod.armour.option.durability.percentage=Prosentti
+sol_client.mod.armour.option.durability.fraction=Murto-osa
+
+sol_client.mod.armour.option.armour=Näytä panssari
+sol_client.mod.armour.option.hand=Näytä Käsi
+sol_client.mod.armour.option.horizontal=Vaakasuora
+
+sol_client.mod.timers.name=Ajastimet
+sol_client.mod.timers.description=Näytä ajastimet pelin-sisäisiin tapahtumiin.
+
+sol_client.mod.timers.option.alignment=Linjaus
+sol_client.mod.timers.option.icon=Kuvake
+sol_client.mod.timers.option.nameColour=Etiketin väri
+sol_client.mod.timers.option.timeColour=Ajan väri
+
+sol_client.mod.chat.name=Chat
+sol_client.mod.chat.description=Mukauta pelinsisäistä chat keskustelua.
+
+sol_client.mod.chat.peek=Chat keskustelun kurkistaminen
+sol_client.mod.chat.option.preventClose=Estä Sulkeminen
+sol_client.mod.chat.option.smooth=Viestin Animaatio
+sol_client.mod.chat.option.infiniteChat=Ääretön Chat Keskustelu
+sol_client.mod.chat.option.peekKey=Kurkistamisen näppäin
+
+sol_client.mod.chat.option.visibility=Näkyvyys
+sol_client.mod.chat.option.visibility.shown=Näytetään
+sol_client.mod.chat.option.visibility.commands=Vain Komennot
+sol_client.mod.chat.option.visibility.hidden=Piilotettu
+
+sol_client.mod.chat.option.defaultTextColour=Tekstin Oletusväri
+sol_client.mod.chat.option.colours=Värit
+sol_client.mod.chat.option.width=Leveys
+sol_client.mod.chat.option.closedHeight=Suljettu korkeus
+sol_client.mod.chat.option.openHeight=Avoin korkeus
+sol_client.mod.chat.option.links=Linkit
+sol_client.mod.chat.option.promptLinks=Pikalinkit
+sol_client.mod.chat.option.chatFilter=Ota chat-suodatin käyttöön
+sol_client.mod.chat.option.filteredWordsContent=Suodatetut Sanat
+
+sol_client.mod.tab_list.name=Tab List
```

Comment:
Still in English. Not sure if you'd want to translate this though.

Reply:
I don't want to, 100% this is more understandable